scripts/tools/tools-schema.json

```diff
@@ -0,0 +1,82 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "JSON Schema for AsyncAPI tool discovery file.", 
+	"type": "object",
+	"required": [
+		"title",
+		"description",
+		"maintainers",
+		"links",
+		"filters"
+	],
+	"properties": {
+		"title": {
+			"type": "string",
+            "description": "Human-readable name of the tool that will be visible to people in the list of tools",
+            "examples" : [ "AsyncAPI Generator", "Cupid" ]  
+		},
+		"description": {
+			"type": "string"
+		},
+		"maintainers": {
+			"type": "array",
+			"items":{
+				"type": "string"
+			}
+
+		},
+		"links": {
+			"type": "object",
+			"required": [
+				"websiteUrl",
+				"docsUrl",
+				"iconUrl"
```

Comment:
Do we need all links? I don't remember discussion? 😅

Reply:
We do need these links but I think these should not be kept as required as there can be tools without having official website. 
Also, I'm not sure about `iconUrl` whether we should keep this or not?  Refer to this [comment](https://github.com/asyncapi/website/issues/383#issuecomment-1244071584) and give your opinion. cc: @derberg

Reply:
Hmm. Not every tool has icon, website or docs - icon and website shouldn't be in my opinion required. Docs also, and if someone won't define it we can use Readme.md of given tool as docsURL? What do you think? cc @derberg

Reply:
yeah, I think all should be optional,

the only thing we are sure of is repo url

Reply:
so yeah, no fallbacks, just `View on GitHub` will be enough

Reply:
and do we need `iconUrl` for tools, because as recent designs shown by Missy in comment https://github.com/asyncapi/website/issues/383#issuecomment-1243881982, I liked the right side design more (not having tools avatar). What's your view on this?

Reply:
> so yeah, no fallbacks, just View on GitHub will be enough

Ok.

Reply:
makes sense to remove it, 90% of cases there will be no icons anyway